wordpress-mobile/WordPress-FluxC-Android
# Replace Stetho with Flipper

Stetho has been deprecated and it's crashing the app when Chrome dev tools are connected. Flipper is the new Stetho, it comes with a Mac binary as a debugging tool.

I only added the layout inspector and network plugins, we can add more (leak canary, databases and preferences could be great additions).

To test:
- Install the desktop app: https://www.facebook.com/fbflipper/public/mac and run it.
- Run the example (or instaflux) app.
- Make sure the Flipper app connects to the device/emulator.
- Enable the layout inspector and the network plugin in the desktop app.
- Make sure network requests are logged in Flipper, and make sure you can inspect the layout.

More about Flipper here: https://fbflipper.com/
